.github/workflows/build.yml

```diff
@@ -0,0 +1,45 @@
+name: build
+
+on: [push, workflow_dispatch]
+
+env:
+  BUNDLE_JOBS: 4
+  BUNDLE_RETRY: 4
+  BUNDLE_PATH: vendor/bundle
+jobs:
+
+  build:
+    runs-on: ubuntu-latest
+    permissions:
+      packages: write
+      contents: read
+      issues: read
+      checks: write
+      pull-requests: write
+    defaults:
+      run:
+        shell: bash
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v1
+
+      - name: Set up Ruby 2.7.0
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: 2.7.0 
+
+      - name: bundler version
+        run:  bundler -v
+        working-directory: /home/runner/work/X12/X12
+
```

Comment:
It seems like we should be running the specs as a step here to prevent possibly publishing a broken gem.